packages/fsml/fsml.0.2a/opam

```diff
@@ -0,0 +1,29 @@
+opam-version: "2.0"
+synopsis:
+  "A library for describing and describing synchronous finite state machines"
+maintainer: ["Jocelyn Sérot <jocelyn.serot@uca.fr>"]
+authors: ["Jocelyn Sérot <jocelyn.serot@uca.fr>"]
+license: "MIT"
+homepage: "http://github.com/jserot/fsml"
+doc: "http://jserot.github.io/fsml"
+bug-reports: "jocelyn.serot@uca.fr"
+depends: [
+  "dune" {>= "2.6"}
+  "menhir" {>= "20200624"}
+  "ocaml" {>= "4.10" & < "4.11"}
+]
+build: [
+  ["dune" "subst"] {pinned}
+  [
+    "dune"
+    "build"
+    "-p"
+    name
+    "-j"
+    jobs
+    "@install"
+    "@runtest" {with-test}
+    "@doc" {with-doc}
+  ]
+]
+dev-repo: "git+https://github.com/jserot/fsml.git"
```

Comment:
The problem is that here there should be something of the following form:
```
url {
  src: "URL_to_SOURCE_TARBALL"
  checksum: [
    "md5=MD5_of_SOURCE_TARBALL"
    ]
}
```
Usually opam-publish adds it for you, but in this case it does not seem to pick it up.

Reply:
Isn't the line : 
```
dev-repo: "git+https://github.com/jserot/fsml.git"
```
in the opam file supposed to do this ?
The last package i published (`rfsmc`) only has it.
Maybe updgrading `opam-publish` could solve the pb ?

```
$ opam info opam-publish

<><> opam-publish: information on all versions ><><><><><><><><><><><><><><>  🐫 
name                   opam-publish
all-installed-versions 2.0.1 [4.06.0]
all-versions           0.2.0  0.2.1  0.3.0  0.3.0+transition  0.3.5  2.0.0~beta  2.0.0  2.0.1  2.0.2
```
 Having to manually add a link to a tar ball and computing a MD5 is a pain - and it is exactly what opam-publish is supposed to to, isn't it ?

Reply:
If there is only the `dev-repo` It must have been merged by mistake. Opam needs the source fields to find the package. Usually opam-publish adds it automatically, I don’t know why it is not doing it

Reply:
I have checked, `rfsm` contains the url: https://github.com/ocaml/opam-repository/pull/14847/files#diff-088e1fe6b78480d7631b40eb8e22c5eeR34

Reply:
Ok, I've added the `url` section to the opam file.
I'll try later to upgrade `opam-publish` to check wheter it solves the pb.